apps/site/util/imageUtils.ts

```diff
@@ -0,0 +1,13 @@
+export const isSvgImage = (src: string) => {
+  let isSvg = false;
+
+  if (src.includes('.svg')) {
```

Comment:
```suggestion
  if (src.endsWith('.svg')) {
```

Reply:
This is a pre-check before converting to URL, `endsWith` causes problems because there is a query parameter at the end of some image

https://github.com/nodejs/nodejs.org/blob/main/apps/site/pages/en/about/previous-releases.mdx?plain=1#L15

Reply:
Please do not instantiate a URL :D